crates/biome_js_type_info/tests/snapshots/infer_type_of_function_with_destructured_arguments.snap

```diff
@@ -14,20 +15,7 @@ function destruct(
 sync Function "destruct" {
   accepts: {
     params: [
-      required (unnamed): Object {
-        prototype: No prototype
-        members: {TypeMembers(
-          Property(
-            [a, required]
-            Type(number)
-          )
-          Property(
-            [b, required]
-            Type(string)
-          )
-        )}
-      } (bindings:
-        a:Object {
+        required (unnamed): Object {
```

Comment:
Is it intended to have an extra indent here? It's just a styling for debug output though